google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/apis/CloudApiMavenService.java

```diff
@@ -45,13 +44,9 @@ static CloudApiMavenService getInstance() {
    * @return returns the {@link Version versions} of the BOMs
    */
   List<Version> getBomVersions() {
-    List<RemoteRepository> remoteRepositories =
-        ImmutableList.of(ArtifactRepositoryManager.MAVEN_CENTRAL_REPOSITORY);
```

Comment:
is MAVEN_CENTRAL_REPOSITORY still used after this? wondering how it knows which remote repo to access with this update

Reply:
They use two central repos - MAVEN_CENTRAL_REPOSITORY and JBOSS_COMMUNITY_REPOSITORY in this constructor. As an alternative, we may create Maven central repo directly from URL ourselves and use it instead so we can have more control, wdyt?

Reply:
> we may create Maven central repo directly from URL ourselves and use it instead so we can have more control, wdyt?

Could you elaborate on that?

Reply:
Sure, they create the repository like that in their code:
```java
private static final RemoteRepository MAVEN_CENTRAL_REPOSITORY = createRemoteRepository(
    "central", "http://repo1.maven.org/maven2/"
  );
```

we can take this approach and manually create the same repository in our code without relying on their selection of public repositories for `ArtifactRepositoryManager`

Reply:
ah. I just looked at the code - so by default they use maven central and jboss..I think we want to exclude jboss.

so yeah - what you suggested sounds good to me